src/Features/CSharp/Portable/UseSimpleUsingStatement/UseSimpleUsingStatementCodeFixProvider.cs

```diff
@@ -84,11 +84,25 @@ private static IEnumerable<StatementSyntax> Expand(UsingStatementSyntax usingSta
             var result = new List<StatementSyntax>();
             Expand(result, usingStatement);
 
+            var decendentTrivia = usingStatement.DescendantTrivia().ToList();
+            var comments = new List<SyntaxTrivia>();
+
+            decendentTrivia.ForEach(x =>
+            {
+                if (x.Kind() == SyntaxKind.SingleLineCommentTrivia || x.Kind() == SyntaxKind.MultiLineCommentTrivia)
+                {
+                    comments.Add(x);
+                    comments.Add(SyntaxFactory.EndOfLine(Environment.NewLine));
+                }
+            });
+
             for (int i = 0, n = result.Count; i < n; i++)
             {
                 result[i] = result[i].WithAdditionalAnnotations(Formatter.Annotation);
             }
 
+            result[0] = result[0].WithLeadingTrivia(comments);
+
             return result;
         }
```

Comment:
We'd definitely need tests for this change.  They can be added in https://github.com/dotnet/roslyn/blob/master/src/EditorFeatures/CSharpTest/UseSimpleUsingStatement/UseSimpleUsingStatementTests.cs